src/Tizen.NUI/src/public/BaseComponents/ViewBindableProperty.cs

```diff
@@ -573,6 +573,21 @@ public partial class View
             return view.IsKeyboardFocusable();
         });
 
+        /// <summary>
+        /// FocusableChildrenProperty
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public static readonly BindableProperty FocusableChildrenProperty = BindableProperty.Create(nameof(FocusableChildren), typeof(bool), typeof(View), false, propertyChanged: (bindable, oldValue, newValue) =>
```

Comment:
same as above, could you check the default value is `false`? different from the comment?